VEDA_OS01/utils.py

```diff
@@ -121,4 +123,76 @@ def parse(self, stream, media_type=None, parser_context=None):
         """
         Simply return a string representing the body of the request.
         """
-        return stream.read()
\ No newline at end of file
+        return stream.read()
+
+
+def get_or_create_course(course_id, course_hex=None, create_course_override=False):
```

Comment:
Update the name of `course_hex` into `studio_hex`

Reply:
Let's make `course_hex` do not have a default and required!

Reply:
👍